test/OrchardCore.Tests/Html/HtmlSanitizerTests.cs

```diff
@@ -14,6 +14,7 @@ public class HtmlSanitizerTests
         [InlineData("<script>alert('xss')</script><div onload=\"alert('xss')\">Test<img src=\"test.gif\" style=\"background-image: url(javascript:alert('xss')); margin: 10px\"></div>", "<div>Test<img src=\"test.gif\" style=\"margin: 10px\"></div>")]
         [InlineData("<IMG SRC=javascript:alert(\"XSS\")>", @"<img>")]
         [InlineData("<a href=\"javascript: alert('xss')\">Click me</a>", @"<a>Click me</a>")]
+        [InlineData("<a href=\"[locale 'en']javascript: alert('xss')[/locale]\">Click me</a>", @"<a>Click me</a>")]
```

Comment:
Interesting so it strips everything. cool

Reply:
Now I'm wondering if it strips the shortcode.. will it work ?